### PR TITLE
Branding

### PR DESCRIPTION
**Pull request by KimBurgess**
Minor updates to setup scripts to reference current repo addresses and match user generation to new domains.

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -50,7 +50,7 @@ then
 fi
 
 ENGINE_DOMAIN=${ENGINE_DOMAIN:="localhost:8080"}
-ENGINE_EMAIL=${ENGINE_EMAIL:="support@aca.im"}
+ENGINE_EMAIL=${ENGINE_EMAIL:="support@place.technology"}
```
Comment (caspiano) [inline install]:
```suggestion
 ENGINE_EMAIL=${ENGINE_EMAIL:="support@place.tech"}
```

File: install [hunk 2]
```diff
@@ -85,7 +85,7 @@ then
     echo "=== Setting up sentry..."
     ./scripts/setup-sentry.sh
     echo -e "\n=== Sentry setup complete. Login to http://$ENGINE_DOMAIN on port 8989 with"
-    echo "support@aca.im:development"
+    echo "support@place.technology:development"
```
Comment (caspiano) [inline install]:
```suggestion
    echo "support@place.tech:development"
```

File: restart [hunk 3]
```diff
@@ -62,7 +62,7 @@ printf "=== Pulling Drivers..."
 git clone "https://github.com/placeos/drivers" "drivers" 2> /dev/null || git -C "drivers" pull
 
 ENGINE_DOMAIN=${ENGINE_DOMAIN:="localhost:8080"}
-ENGINE_EMAIL=${ENGINE_EMAIL:="support@aca.im"}
+ENGINE_EMAIL=${ENGINE_EMAIL:="support@place.technology"}
```
Comment (caspiano) [inline restart]:
```suggestion
ENGINE_EMAIL=${ENGINE_EMAIL:="support@place.tech"}
```

File: scripts/setup-sentry.sh [hunk 4]
```diff
@@ -42,7 +42,7 @@ docker-compose -f ./docker-compose.yml -f ./compose-files/sentry/docker-compose.
   -e SENTRY_DB_PASSWORD=test \
   -e SENTRY_REDIS_HOST=redis \
   -e SENTRY_SINGLE_ORGANIZATION=true \
-  sentry createuser --email=support@aca.im --password=test --superuser --no-input
+  sentry createuser --email=support@place.technology --password=test --superuser --no-input
```
Comment (caspiano) [inline scripts/setup-sentry.sh]:
```suggestion
  sentry createuser --email=support@place.tech --password=test --superuser --no-input
```

File: scripts/setup-sentry.sh [hunk 5]
```diff
@@ -61,5 +61,5 @@ docker-compose -f docker-compose.yml -f compose-files/sentry/docker-compose.yml
     up -d sentry sentry-cron sentry-worker
 
 printf -- "--------------------------\n\n"
-printf "You should now be able to login to localhost:8989 as support@aca.im : test\n"
+printf "You should now be able to login to localhost:8989 as support@place.technology : test\n"
```
Comment (caspiano) [inline scripts/setup-sentry.sh]:
```suggestion
printf "You should now be able to login to localhost:8989 as support@place.tech : test\n"
```

**Comment by KimBurgess** [comment]
Can do. That domain alias should be active now too so addresses at both are valid.

Ideally, it would be nice to allow user entry for the default account on the dev env (maybe default to what can be pulled from `git config user.email`) and generate a random pass, but this works for initial testing purposes.

**Pull request merged**